docker-compose/multi/multi-nodes.yml

```diff
@@ -0,0 +1,129 @@
+version: "3.2"
+services:
+  master-api:
+    build:
+      context: ../../api
+    environment:
+      RPC_HOST: master-chain      
+      RPC_PORT: ${RPC_PORT}
+      RPC_PASSWORD: ${RPC_PASSWORD}
+      PORT: ${API_PORT}
+      ORGANIZATION: ${ORGANIZATION}
+      ORGANIZATION_VAULT_SECRET: ${ORGANIZATION_VAULT_SECRET}
+      ROOT_SECRET: asdf
+      LOG_LEVEL: ${LOG_LEVEL}
+      PRETTY_PRINT: ${PRETTY_PRINT}
+    links:
+      - master-chain
+    volumes:
+      - ../../api/src:/home/node/src
+      - ../../api/dist:/home/node/dist
+    command: ["npm", "run", "watch"]
+    ports:
+      - 8000:${API_PORT}
+      - "8085:8085"
+
+  master-chain:
+    # image: trubudget/blockchain:latest
+    build:
+      context: ../../blockchain
+    environment:
+      RPC_PORT: ${RPC_PORT}
+      RPC_PASSWORD: ${RPC_PASSWORD}
+      ORGANIZATION: ${ORGANIZATION}
+      EXTERNAL_IP: ${EXTERNAL_IP}
+    command: ["npm", "run", "watch"]
+    volumes:
+      - ../../blockchain/src:/home/node/src
+    #   - /tmp/bcMasterVolume:/root/.multichain
+    ports:
+      - "7447:7447"
+
+  # frontend cannot be hot-loaded, container contains only built application, no source files 
+  master-frontend:
+    image: trubudget/frontend:latest
+    environment:
+      PROD_API_HOST: master-api
+      PROD_API_PORT: ${API_PORT}
+      TEST_API_HOST: master-api
+      TEST_API_PORT: ${API_PORT}
+      # EXPORT_HOST: export
+      # EXPORT_PORT: 8888
+    depends_on:
+      - master-api
+    volumes:
```

Comment:
@SamuelPull Volumes don't need to be there. Built, compiles and packed image is used. No change in src folder is reflected to the frontend's page.